packages/web/src/common/store/cache/tracks/utils/retrieveTracks.ts

```diff
@@ -62,67 +62,63 @@ export function* retrieveTrackByHandleAndSlug({
 
   // Check if this is the first fetch after server side rendering the track page
   const isInitialFetchAfterSsr = yield* select(getIsInitialFetchAfterSsr)
-  const tracks = (yield* call(
-    // @ts-ignore retrieve should be refactored to ts first
-    retrieve,
-    {
-      ids: [permalink],
-      selectFromCache: function* (permalinks: string[]) {
-        const track = yield* select(getTracksSelector, {
-          permalinks
+  // @ts-ignore string IDs don't play well with the current retrieve typing
```

Comment:
punting on this til later since it's a weird special case

this is the only use of string ids in the cache it seems